src/Libraries/Lang/Lang.php

```diff
@@ -54,25 +55,41 @@ class Lang
      * Instance of Lang
      * @var \Quantum\Libraries\Lang\Lang
      */
-    private static $langInstance = null;
+    private static $instance = null;
 
     /**
      * GetInstance
-     * @return \Quantum\Libraries\Lang\Lang|null
+     * @return \Quantum\Libraries\Lang\Lang
      */
-    public static function getInstance(): ?Lang
+    public static function getInstance(int $langSegmentIndex = 1): ?Lang
```

Comment:
getInstance() should never return null, the return type declaration is not correct